packages/interface/src/components/fullscreen-mode/test/index.js

```diff
@@ -1,76 +1,55 @@
 /**
  * External dependencies
  */
-import { act, create } from 'react-test-renderer';
+import { render } from '@testing-library/react';
 /**
  * Internal dependencies
  */
 import FullscreenMode from '..';
 
 describe( 'FullscreenMode', () => {
 	it( 'fullscreen mode to be added to document body when active', () => {
-		act( () => {
-			create( <FullscreenMode isActive={ true } /> );
-		} );
-		expect( document.body.classList.contains( 'is-fullscreen-mode' ) ).toBe(
-			true
-		);
+		const { baseElement } = render( <FullscreenMode isActive={ true } /> );
```

Comment:
I know this was here before, but we could make it less verbose since `={ true }` is redundant:

```suggestion
		const { baseElement } = render( <FullscreenMode isActive /> );
```